Makefile

```diff
@@ -28,7 +28,7 @@ all: manager readinessServer
 export_env_vars:
 export NAMESPACE = openshift-storage
 export ADDON_NAME = ocs-converged
-export SOP_ENDPOINT = "https://red-hat-storage.github.io/ocs-sop/sop/OSD/{{ .GroupLabels.alertname }}.html"
+export SOP_ENDPOINT = https://red-hat-storage.github.io/ocs-sop/sop/OSD/{{ .GroupLabels.alertname }}.html
```

Comment:
To me, it seems like the quotes are necessary as there are whitespaces in the value. But I might be wrong. 
Was this change verified?

Reply:
I have verified the change by running:
```
$ make install
...
$ make run
...
^C
$ oc get secret/managed-ocs-alertmanager-config-secret -o json | jq -r .data.\"alertmanager.yaml\" | base64 -d
route:
  receiver: pagerduty
  routes:
  - group_wait: 30s
    group_interval: 5m
    repeat_interval: 4h
    receiver: pagerduty
    group_by:
    - alertname
    match:
      severity: critical
  - group_wait: 5m
    group_interval: 5m
    repeat_interval: 5m
    receiver: DeadMansSnitch
    match:
      alertname: DeadMansSnitch
receivers:
- name: pagerduty
  pagerduty_configs:
  - service_key: test-key
    details:
      SOP: https://red-hat-storage.github.io/ocs-sop/sop/OSD/{{ .GroupLabels.alertname }}.html
- name: DeadMansSnitch
  webhook_configs:
  - url: https://test-url
 ```